projects/1.16/assets/pams-harvestcraft-2-food-extended/pamhc2foodextended/lang/zh_cn.json

```
@@ -519,34 +528,36 @@
   "item.pamhc2foodextended.veggiestirfryitem": "炒蔬菜",
   "item.pamhc2foodextended.veggiestripsitem": "蔬菜条",
   "item.pamhc2foodextended.vindalooitem": "辣味咖喱肉",
-  "item.pamhc2foodextended.walnutraisinbreaditem": "胡桃葡萄干面包",
+  "item.pamhc2foodextended.walnutraisinbreaditem": "核桃葡萄干面包",
   "item.pamhc2foodextended.weekendpicnicitem": "周末野餐套餐",
   "item.pamhc2foodextended.wontonsoupitem": "云吞汤",
   "item.pamhc2foodextended.yorkshirepuddingitem": "约克郡布丁",
   "item.pamhc2foodextended.zeppoleitem": "炸面饼圈",
   "item.pamhc2foodextended.zestyzucchiniitem": "烤西葫芦通心粉",
   "item.pamhc2foodextended.zucchinibakeitem": "蔬菜奶酪烤西葫芦",
   "item.pamhc2foodextended.zucchinifriesitem": "炸西葫芦条",
-  "item.pamhc2foodextended.fajitaburritoitem": "墨西哥烤肉玉米卷",
+  "item.pamhc2foodextended.fajitaburritoitem": "墨西哥烤肉卷",
+  
   "item.pamhc2foodextended.cookedgroundbeefitem": "熟牛肉碎",
   "item.pamhc2foodextended.cookedgroundchickenitem": "熟鸡肉碎",
   "item.pamhc2foodextended.cookedgroundfishitem": "熟鱼肉碎",
   "item.pamhc2foodextended.cookedgroundmuttonitem": "熟羊肉碎",
   "item.pamhc2foodextended.cookedgroundporkitem": "熟猪肉碎",
   "item.pamhc2foodextended.cookedgroundrabbititem": "熟兔肉碎",
+  
   "item.pamhc2foodextended.avocadotoastitem": "牛油果吐司",
-  "item.pamhc2foodextended.beancornmealitem": "玉米烤豆子",
+  "item.pamhc2foodextended.beancornmealitem": "玉米烩豆",
   "item.pamhc2foodextended.jalapenoburgeritem": "辣椒牛肉汉堡",
   "item.pamhc2foodextended.molasseschickenitem": "糖蜜炖鸡",

```

```suggestion
  "item.pamhc2foodextended.molasseschickenitem": "蜜汁鸡肉",
```
这里用的是MC百科给出的译名。以及`炖鸡`？看着不太像，而且就我在Bing上看到的食谱，都是`roast` `bake` `grill`之类的，没有`stew`。至于`braised`，倒是有Braised Chicken with Pomegranate Molasses。